_timeline/2016-08-21-Built-Our-First-Nodes.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Built our First Mesh Nodes
+date: 2016-08-21
+---
+
+Toronto Mesh hosted "[Build a Mesh Node](https://tomesh.net/2016-08-21/build-a-node/)," a free public workshop to get members up and running with a our first working node.
```

Comment:
I think we don't need the quotes around `Build a Mesh Node` and we should capitalize the _A_ for consistency.